fundingmanager.go

```diff
@@ -1386,44 +1445,59 @@ func (f *fundingManager) waitForFundingConfirmation(completeChan *channeldb.Open
 		return
 	}
 
-	// Now that the funding transaction has the required number of
-	// confirmations, we send the fundingLocked message to the peer.
-	f.sendFundingLockedAndAnnounceChannel(completeChan, &shortChanID)
+	confChan <- &shortChanID
```

Comment:
In order to avoid possible future deadlocks on shutdown, this should be converted to instead be a conditional send:
```go
select {
   case confChan <- &shortChanID:
   case <-f.quit:
        return
}
```